lib/batch.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const ApiGroup = require('./api-group');
+
+class Batch extends ApiGroup {
+  constructor(options) {
+    const genericTypes = [
+      'scheduledjobs'
+    ];
+    options = Object.assign({}, options, {
+      path: 'apis/batch',
+      version: options.version || 'v2alpha1',
```

Comment:
Really good question, I was thinking about that as well. 

I guess it makes sense to have the "conservative" default here, anyone that wants alpha/beta stuff should do so knowingly.